Userland/Applications/DisplaySettings/FontSettingsWidget.h

```diff
@@ -20,6 +23,11 @@ class FontSettingsWidget : public GUI::Widget {
 
 private:
     FontSettingsWidget();
+
+    static void update_label_with_font(RefPtr<GUI::Label>, RefPtr<Gfx::Font> const&);
```

Comment:
This doesn't need to be a static member function, just make it a static function in the cpp file.